.github/workflows/gradle.yml

```diff
@@ -112,6 +124,21 @@ jobs:
         id: version_info
         run: echo version=$(./gradlew properties -q -Prelease.useLastTag=true | grep "version:" | awk '{print $2}') >> "$GITHUB_OUTPUT"
 
+      - name: Publish to NuGet
+        if: github.ref == 'refs/heads/main'
+        env:
+          NUGET_TOKEN: ${{ secrets.NUGET_TOKEN }}
+        run: |
+          ./gradlew mingwX64Binaries
+          ClangSharpPInvokeGenerator -f doordeck-sdk/build/bin/mingwX64/releaseShared/doordeck_headless_sdk_api.h -o doordeck-sdk/build/bin/mingwX64/releaseShared/doordeck_headless_sdk/DoordeckHeadlessBindings.cs -n DoordeckHeadlessSDK
+          sed -i '' 's/_\$$/_/g' doordeck-sdk/build/bin/mingwX64/releaseShared/doordeck_headless_sdk/DoordeckHeadlessBindings.cs
+          sed -i '' 's/\$//g' doordeck-sdk/build/bin/mingwX64/releaseShared/doordeck_headless_sdk/DoordeckHeadlessBindings.cs
+          sed -i '' '/\[NativeTypeName/d' doordeck-sdk/build/bin/mingwX64/releaseShared/doordeck_headless_sdk/DoordeckHeadlessBindings.cs
+          sed -i '' 's/\[DllImport(""/\[DllImport("doordeck_headless_sdk.dll"/g' doordeck-sdk/build/bin/mingwX64/releaseShared/doordeck_headless_sdk/DoordeckHeadlessBindings.cs
```

Comment:
You could combine these if you wanted, you can have multiple `-e` regexes with sed, e.g.
```
sed -i '' \
    -e 's/_\$$/_/g' \
    -e 's/\$//g' \
    -e '/\[NativeTypeName/d' \
    -e 's/\[DllImport(""/\[DllImport("doordeck_headless_sdk.dll"/g' \
    "doordeck-sdk/build/bin/mingwX64/releaseShared/doordeck_headless_sdk/DoordeckHeadlessBindings.cs"
```

Reply:
Thank you, it looks better now!